pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -893,6 +893,11 @@ func possibleGuestSize(disk api.Disk) (int64, bool) {
 		log.DefaultLogger().Reason(err).Error("Failed to parse filesystem overhead as float")
 		return 0, false
 	}
+	if filesystemOverhead < 0 || filesystemOverhead > 1 {
+		log.DefaultLogger().Errorf("Invalid filesystem overhead %f (must be between 0 and 1)", filesystemOverhead)
+		return 0, false
+	}
+
```

Comment:
Note: This validation was not used before. Why do we handle the validation so deep in the stack?

Reply:
Right before this we parse it from a string to a float, and I figured that it's easier to validate it when it is a float.

This way we reject out-of-range values in the same place we reject non-numerical values.

I can move it elsewhere if that's preferable.

Reply:
I don't see a reason for the validation to be here (generally). I appreciate if we have a look at it in the follow up